site/_components/card.njk

```diff
@@ -2,33 +2,34 @@
   Renders a clickable card.
 
   Arguments:
-    data: List of objects containing the following attributes.
-      - eyebrow (optional): Card eyebrow.
-      - title: Card title.
-      - body: Card body content.
-      - source (optional): Object containing the information source.
-        - logo: The source logo.
-        - name: The source name. i.e. Google Dev Blog.
-      - url: Card URL.
-    modifiers (optional): Object with modifier styles for the card.
-      - custom {String}: Custom modifier styles for the card.
-      - rounded {Boolean}: Rounded border style. Defaults to true.
-    idPrefix (optional): Additional identifier to prevent repeated IDs if multiple cards share the same title.
+    @param {object} data: List of objects containing the following attributes.
+    @param {string} eyebrow (optional): Card eyebrow.
+    @param {string} title: Card title.
+    @param {string} body: Card body content.
+    @param {object} source (optional): Object containing the information source.
+    @param {string} logo: The source logo.
+    @param {string} name: The source name. i.e. Google Dev Blog.
+    @param {string} url: Card URL.
+    @param {string} cta (optional): Text to use in footer when no source is provided.
```

Comment:
```suggestion
    @param {string} data.eyebrow (optional): Card eyebrow.
    @param {string} data.title: Card title.
    @param {string} data.body: Card body content.
    @param {object} data.source (optional): Object containing the information source.
    @param {string} data.logo: The source logo.
    @param {string} data.name: The source name. i.e. Google Dev Blog.
    @param {string} data.url: Card URL.
    @param {string} data.cta (optional): Text to use in footer when no source is provided.
```

I'm all for updating this to follow the other doc patterns we have. In this case, we should include that these are properties of `data`